src/style.css

```diff
@@ -0,0 +1,407 @@
+/* width */
+::-webkit-scrollbar {
+  width: 3px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  background: #f1f1f1;
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: #888;
+}
+
+:root {
+  --darkBg: #17161b;
+  --lightBg: #1d1d1f;
+  --redBg: #ea1c24;
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+  background: #555;
+}
+
+@font-face {
+  font-family: 'Inter';
+  src: url('./assets/fonts/Inter/Inter-VariableFont_slnt\,wght.ttf');
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+}
+
+body {
+  background-color: var(--darkBg);
+  color: #fff;
+  overflow-x: hidden;
+  display: flex;
+  flex-direction: column;
+}
+
+main {
+  position: relative;
+  background-image: url('./assets/images/bg-capstone.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+main::before {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  pointer-events: none; /* Permite que los eventos del mouse pasen a través del pseudo-elemento */
+}
+
+.material-symbols-outlined {
+  font-variation-settings: 'FILL' 0, 'wght' 400, 'GRAD' 0, 'opsz' 48;
+  font-size: 40px;
+  margin-right: 5px;
+}
+
+.nav {
+  display: flex;
+  gap: 20px;
+  align-items: center;
+  padding: 1.1rem 1.5rem;
+  border-bottom: 1px solid #8e8e8e;
+  border-width: thin;
+  background-color: #f1f1f1;
+  color: #000;
+}
+
+.app-logo {
+  width: 15%;
+  display: flex;
+  align-items: center;
+}
+
+.app-logo img {
+  width: 35px;
+  height: 35px;
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+  gap: 2rem;
+}
+
+.app-navigation {
+  flex: 1;
+}
+
+.nav-list a {
+  color: #000;
+  text-decoration: none;
+}
+
+.nav-list-item a {
+  color: #000;
+  text-decoration: none;
+}
+
+.nav-list a:hover {
+  color: var(--redBg);
+}
+
+.profile-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 3px;
+}
+
+.profile {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 2rem;
+  height: 2rem;
+  background-color: #fff;
+  border-radius: 100%;
+}
+
+.profile-arrow-down {
+  color: #fff;
+}
+
+.item-counter {
+  display: flex;
+  margin-top: 10px;
+  align-items: center;
+  justify-content: space-between;
+  padding: 5px 100px;
+}
+
+.counter {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+footer {
+  width: 100%;
+  background-color: #f1f1f1;
+  color: #000;
+  border-top: 1px solid grey;
+  height: 90px;
+  padding: 35px;
+}
+
+/* url('./assets/images/bg-capstone.jpg'); */

```

Comment:
- [ ]  I would be more professional to remove this commented line of code, doing this will make you codebase cleaner. 👍